src/index.d.ts

```diff
@@ -60,6 +71,18 @@ interface getTaskByID {
   startDate: string;
   endDate: string;
   estimeeSpentTime: string;
+  tags: ITagList;
+}
+interface ITagList {
+  __typename?: string;
+  id: string;
+  label: string;
+  color: string;
+}
```

Comment:
Soon you'll disappear... and I love it!

![f6d56508e8f10cce95a752bd1dc38613](https://user-images.githubusercontent.com/19322697/150505761-4dca80c6-1543-4fab-8b31-5c91b9d007a8.jpeg)